integration/flink/src/test/scala/org/apache/carbon/flink/TestCarbonWriter.scala

```diff
@@ -95,9 +96,18 @@ class TestCarbonWriter extends QueryTest {
           throw new UnsupportedOperationException(exception)
       }
 
+      checkAnswer(sql(s"select count(1) from $tableName"), Seq(Row(0)))
+
+      // query with stage input
+      CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_QUERY_STAGE_INPUT, "true")
```

Comment:
can we add one filter query test case also ?

Reply:
added